app/views/home/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>WELCOME BLOGIFY HOMEPAGE</h1>
+<p>This is the index page on this website</p>
```

Comment:
- [ ] You have no user index page. We don't need a home view here. Kindly remove this file and this home folder.